app/models/asset_field_type.rb

```diff
@@ -34,6 +34,7 @@ def data
         'dimensions': dimensions,
         'content_type': asset_content_type,
         'file_size': asset_file_size,
+        'has_thumb': is_image(asset_content_type),
```

Comment:
Again, this is assuming that all image media file should have thumbnails attached, which may not necessarily be the case